src/main/java/com/example/springfirsthw/dto/CreateBookRequestDto.java

```diff
@@ -1,20 +1,25 @@
 package com.example.springfirsthw.dto;
 
+import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
-    @NotNull(message = "can't be null")
+    @NotBlank(message = "can't be blank")
+    @Size(min = 1, max = 50)
     private String title;
-    @NotNull(message = "can't be null")
+    @NotBlank(message = "can't be blank!")
     private String author;
     @NotNull(message = "can`t be null")
+    @ISBN(message = "wrong format of ISBN")
     private String isbn;
-    @NotNull(message = "can`t be null")
-    @Positive(message = "must be more or equals 0")
+    @Positive(message = "must be more than 1")
+    @NotBlank(message = "can`t be blank!")
```

Comment:
It looks something weird to check if BigDecimal is blank. We usually check if string is not blank.
Please, avoid using NotBlank.